test/test_unit_strings.cpp

```diff
@@ -442,6 +442,15 @@ TEST(unitStrings, watthours)
     EXPECT_EQ(str, "GWh*m");
 }
 
+TEST(unitStrings, mm)
+{
+    auto speedUnit = unit_from_string("mm/s");
+    EXPECT_EQ(to_string(speedUnit), "mm/s");
+
+    auto accUnit = unit_from_string("mm/s^2");
+    EXPECT_EQ(to_string(accUnit), "mm/s^2");
```

Comment:
Have you checked if these are the only examples, or is this part of a wider class of regressions that was introduced recently?

For example, `km/s` currently also formats as `kHz*m`, is this also fixed by the change?

Reply:
The cause of this issue is a combination of two things.  The first being a check taking a single base unit away from the unit, and 'm' was first in the list.   so the remaining portions of the unit were then converted in isolation and since 1/s=Hz it also adds any prefix,  by swapping the order to dealing with 's' first it takes care combinations of 'm' and 's'  a bit better.    So as best as I can tell the only types of unit this issue is affecting is combinations of 'meter' and 'second'   so yes it also fixes fixes km/s.    and gray being m^2/s^2 is a bit of quirk as well, that comes from handling meters in this fashion before s^2 so adding that check first solves that issue as well.